Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -69,27 +78,35 @@ There should be several errors in the `colour.kt` file. We miss the
 declarations for colors in the `src/commonMain/kotlin/common.kt`:
 
 ```kotlin
-expect class Color
-
 object Colors
+
+expect class Color
 expect fun Colors.newColor(r: Int, g: Int, b: Int): Color
 expect val Colors.BLACK : Color
 ```
 
+The `expect class`, `expect fun`, and `expect val` declarations in the Kotlin/Common
```

Comment:
The `expect class`, `expect fun`, and `expect val` declarations in the Kotlin/Common
code, instructs the compiler that we'd like to use these declarations in our Kotlin/Common
code, but we are not able to provide implementations. Implementations are provided via the
`actual` keyword independently for the JVM and JS targets (in general),
[Kotlin/Native](https://kotlinlang.org/docs/reference/native-overview.html) can
also be [used](https://kotlinlang.org/docs/tutorials/native/mpp-ios-android.html)
to target iOS and other platforms).